### ISSUE TITLE
This is super cool

### ISSUE THREAD

**Issue by rgwood** [JeremyKuhne/WInterop#9]
I just wanted to say that WInterop is really great. I'm especially impressed by the thorough, well-documented Petzold examples.

It's remarkably difficult to learn Win32 programming today, because most of the resources are 20+ years old and beginners have to spend a lot of time distinguishing outdated info from still-relevant info. Projects like WInterop are essential to keep this knowledge up-to-date and accessible via modern toolchains.

In short, I feel confident saying that WInterop is Good Software™. Thanks for making it.
